index.bs

```diff
@@ -529,6 +530,9 @@ permission to access any devices so the page must first call
 </pre>
 </div>
 
+Methods defined in this specification typically complete asynchronously, queuing
+work on the <dfn>USB task source</dfn>.
```

Comment:
Is this just a named instance of a https://html.spec.whatwg.org/multipage/webappapis.html#task-source concept?  Link to https://html.spec.whatwg.org/multipage/webappapis.html#task-source?